fabric-network/test/impl/query/queryhandlers.js

```diff
@@ -90,7 +90,7 @@ describe('QueryHandlers', () => {
 			try {
 				await handler.evaluate(query);
 			} catch (error) {
-				expect(error.message).to.contain('peer1-400');
+				expect(error.message).to.contain('peer error response: peer1-400');
```

Comment:
I'm not convinced it is helpful to include the complete message in the unit test. The test verifies the behaviour that the error message includes the response returned by the peer. Having the test check the rest of the error message text just increases coupling between test and implementation code, making it more work to change.

I think unit tests are good for verifying observable behaviour. Quality of message (and log) text is better judged my human review rather than through unit testing.